etc/config/ylc.defults.properties

```diff
@@ -0,0 +1,5 @@
+compilers=/opt/compiler-explorer/ygen/bin/ylc
```

Comment:
AH You have misspelled `ylc.defaults.properties` - which is why you aren't seeing any compilers when you run locally. Rename this file and it will probably start working locally.

Reply:
Oh thank you! Now i am finally able to test locally